lib/KSwitch.vue

```diff
@@ -41,6 +42,13 @@
   export default {
     name: 'KSwitch',
     props: {
+      /**
+       *arialabelledBy used for complex layout
+       */
```

Comment:
It would be better to have a message like this:
```suggestion
      /*
       * Specifies the ID of an element that labels this switch component.
       * Useful for accessibility to provide a label through an external element when `label` is empty
       */
```

Reply:
Okay @AlexVelezLl,  I'll do the changes.